prow/plugins.yaml

```diff
@@ -37,6 +41,9 @@ approve:
   - kubernetes/steering
   - kubernetes/test-infra
   implicit_self_approve: true
+- repos:
+  - kubernetes-sig-testing
+  implicit_self_approve: true
```

Comment:
I'm thinking that this config option should be replaced with an `explicit_self_approve` option in order to get the opposite default behavior. It seems like implicit_self_approve is the common case and changing the default behavior would remove most of this config. I'll open an issue.